### DISCUSSION TITLE
SPI/non-SPI and CH340C?

### DISCUSSION THREAD

**Discussion by DariBer** [awawa-dev/HyperSerialEsp8266#12] [Q&A]
I have 2 questions regarding your firmware HyperSerialEsp8266.

Looking at the Pinout description,  it's deiiefrentf if the LED strip is SPI or non-SPI. Is a default WS2812B SPI or non-SPI?

In the info it's mention that a ESP8266 d1 mini CH340G would work, but my ESP8266 cd1 mini is CH340C, so will a CH340C chip work instead of a CH340G?

**Comment by awawa-dev** [comment]
CH340C  should work: the differences are minimal in the spec. WS2812B  is non-SPI (neopixel) LED strip.
